draft-tschofenig-rats-psa-token.md

```diff
@@ -608,6 +636,68 @@ A PSA token is always directly signed by the PSA RoT.  Therefore, a PSA
 claims-set ({{sec-psa-claims}}) is never carried in a Detached EAT bundle
 ({{Section 5 of EAT}}).
 
+### Freshness Model
+
+The PSA Token supports the freshness models for attestation Evidence based on
+nonces and epoch handles ({{Section 10.2 and Section 10.3 of RFC9334}}) using
+the `nonce` claim to convey the nonce or epoch handle supplied by the Verifier.
+No further assumption on the specific remote attestation protocol is made.
+
+Note that use of epoch handles is constrained by the type restrictions imposed by the `eat_nonce` syntax.
+For use in PSA tokens, it must be possible to encode the epoch handle as an opaque binary string between 8 and 64 octets.
+
+## Profile LM
+{: #sec-ecdsa-profile}
+
+This profile is suitable for devices using TrustedFirmware-M {{TF-M}} built with profile Large, Medium and Medium-ARoT-less.
+
+{{ecdsa-profile}} presents a concise view of the requirements.
+
+The value of the `eat_profile` MUST be `tag:psacertified.org,2023:psa#lm+es`.
```

Comment:
I am confused about the `#lm+es` part, I was expecting `#a+es`.

Reply:
Since they are supposed to map onto TF-M profiles [medium](https://tf-m-user-guide.trustedfirmware.org/configuration/profiles/tfm_profile_medium.html) / [large](https://tf-m-user-guide.trustedfirmware.org/configuration/profiles/tfm_profile_large.html) (and similarly for [small](https://tf-m-user-guide.trustedfirmware.org/configuration/profiles/tfm_profile_small.html)), I thought a more fitting acronym could me "LM", for Large/Medium and "S" for Small... I hold no strong opinions and I am happy to revert.